contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,186 @@
+# Encryption
+
+## Abstract
+
+The scope of this proposal is to ensure that resources can be encrypted at the datastore layer. Encrypting data at REST and via third party vendors is a desired feature but outside the scope of this proposal. Encryption will be optional for any resource but we suspect it will be used for the Secret resource in most cases.
+
+Allowing sensitive data to be encrypted adheres to best practices as well as other requirements such as HIPAA.
+
+How encryption keys are delivered to the machine running the Kubernetes apiserver is of relevance - we assume that the encryption at rest pattern is loosely coupled to how those keys are delivered and secured on disk.
```

Comment:
Building at least the UX for this in the first version allows for expansion to something like secret sharing / key splitting a la https://github.com/hashicorp/vault and https://github.com/cloudflare/redoctober